msdk-id-sirius/pom.xml

```diff
@@ -43,36 +43,36 @@
     <dependency>
       <groupId>de.unijena.bioinf.ms</groupId>
       <artifactId>chemistry_base</artifactId>
-      <version>4</version>
+      <version>4.0</version>
```

Comment:
Please move this version number to a property in pom.xml. I've posted a full example repo for you which both demonstrates the use of a property and how to use slf4j with various logging backends: https://github.com/chhh/slf4j-example